### ISSUE TITLE
Syntax error in `btrfsmaintenance-functions` causes btrfsmaintenance to crash

### ISSUE THREAD

**Issue by lordofpipes** [kdave/btrfsmaintenance#120]
Under certain configurations, the following happens:

```
Jun 06 01:34:46 autumn btrfs-scrub.sh[1041619]: /usr/share/btrfsmaintenance/btrfsmaintenance-functions: line 106: syntax error near unexpected token `fi'
Jun 06 01:34:46 autumn btrfs-scrub.sh[1041619]: /usr/share/btrfsmaintenance/btrfsmaintenance-functions: line 106: `                fi'
Jun 06 01:34:46 autumn btrfs-scrub.sh[1041628]: Running scrub on ...
Jun 06 01:34:46 autumn btrfs-scrub.sh[1041628]: /usr/share/btrfsmaintenance/btrfs-scrub.sh: line 43: run_task: command not found
```

**Comment by MartinX3** [comment]
@lordofpipes I also get this `run_task` error often.
Is it related to this syntax error?

**Comment by kdave** [comment]
Released in 0.5.2, thanks.
